### PR TITLE
Parent user tables crud merge into main

### PR DESCRIPTION
**Pull request by aplasket**
# high level what you did:
- update bill database with addition attributes
- add crud to user_categories
- add crud to user_bills

## Overall Checklist:
- [x] tests written for all methods added
- [x] test coverage above 90%
- [x] RSpec running clean
- [x] ran rails s -- features working properly!
- [x] pulled/merged milestone branch before submitting PR

### CODE REVIEW

**Review by Jredish11 — APPROVED** [review]
awesome work!

File: spec/factories/factories.rb [hunk 1]
```diff
@@ -9,11 +9,103 @@
...
-    bill_number { Faker::Alphanumeric.alphanumeric(number: 4, min_alpha: 1, min_numeric: 3) }
-    text_url { Faker::Internet.url(host: 'https://legiscan.com') }
-    last_action_date { Faker::Date.between(from: 30.days.ago, to: Date.today)  }
-    last_action { Faker::Subscription.status }
-    title { Faker::Hipster.sentence }
+    bill_number { "H" + Faker::Alphanumeric.alphanumeric(number: 4, min_alpha: 0, min_numeric: 4) }
+    status {[0,1,2,3,4].sample}
+    description { Faker::Hipster.sentence }
     state { Faker::Address.state_abbr }
+    texts { [
```
Comment (jcjurado3) [inline spec/factories/factories.rb:16]: Great use of the factory bot!

**Review by jcjurado3 — APPROVED** [review]
Great work! Nice clean code!

File: app/poros/api_bill.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ class ApiBill
               :status,
               :description,
               :sponsors,
-              :text
+              :texts
```
Comment (JCBeldo) [inline app/poros/api_bill.rb:13]: yes, I was gonna say this needed to be updated.  Nice catch :)

**Review by JCBeldo — APPROVED** [review]
Wonderfully written code y'all. 👍

**Pull request merged**